src/main/drivers/adc_stm32f7xx.c

```diff
@@ -47,6 +47,27 @@
 #define ADC1_DMA_STREAM DMA2_Stream4
 #endif
 
+// Copied from stm32f7xx_ll_adc.h
+
+#define VREFINT_CAL_VREF                   ( 3300U)                    /* Analog voltage reference (Vref+) value with which temperature sensor has been calibrated in production (tolerance: +-10 mV) (unit: mV). */
+#define TEMPSENSOR_CAL1_TEMP               (( int32_t)   30)           /* Internal temperature sensor, temperature at which temperature sensor has been calibrated in production for data into TEMPSENSOR_CAL1_ADDR (tolerance: +-5 DegC) (unit: DegC). */
+#define TEMPSENSOR_CAL2_TEMP               (( int32_t)  110)           /* Internal temperature sensor, temperature at which temperature sensor has been calibrated in production for data into TEMPSENSOR_CAL2_ADDR (tolerance: +-5 DegC) (unit: DegC). */
+#define TEMPSENSOR_CAL_VREFANALOG          ( 3300U)                    /* Analog voltage reference (Vref+) voltage with which temperature sensor has been calibrated in production (+-10 mV) (unit: mV). */
+
+// These addresses are incorrectly defined in stm32f7xx_ll_adc.h
+
+#if defined(STM32F745xx) || defined(STM32F746xx)
+// F745xx_F746xx
+#define VREFINT_CAL_ADDR                   ((uint16_t*) (0x1FF0F44A))
```

Comment:
I'd look into defining these as an offset from system memory start

Reply:
May be, but these addresses are given in the data sheets.

Reply:
This constant is defined in LL drivers ...